FHICORC/ViewModels/Certificates/InfoVaccineTextViewModel.cs

```diff
@@ -112,8 +112,15 @@ public void UpdateView()
 
         public void SetAccessibilityTextDate()
         {
-            var vaccinationDate = DateTime.ParseExact(VaccineVaccinationDateValue, "yyyy-MM-dd", CultureInfo.InvariantCulture);
-            VaccineVaccinationDateValueAccessibilityText = string.Format("{0:dd. MMMM yyyy}", vaccinationDate);
+            try
+            {
+                var vaccinationDate = DateTime.ParseExact(VaccineVaccinationDateValue, "yyyy-MM-dd", CultureInfo.InvariantCulture);
+                VaccineVaccinationDateValueAccessibilityText = string.Format("{0:dd. MMMM yyyy}", vaccinationDate);
+            }
+            catch (FormatException)
```

Comment:
Do you want to set `VaccineVaccinationDateValueAccessibilityText` to something in this case?